Dockerfile.local

```diff
@@ -6,9 +6,13 @@ ENV DEBIAN_FRONTEND=noninteractive
 ENV PYTHONUNBUFFERED=1
 
 RUN  apt-get update \
-  && apt-get install -y python3 python3-dev libgmp-dev python3-pip git \
+  && apt-get install -y python3 python3-dev libgmp-dev python3-pip libtool pkg-config git \
   && rm -rf /var/lib/apt/lists/*
 
+WORKDIR /opt
+RUN git clone https://gitee.com/primihub/libzmq.git \
```

Comment:
Why use Gitee instead of GitHub?

Reply:
Dockerfile.local Only used by local build images